report/03_methodology/report.md

```diff
@@ -0,0 +1,221 @@
+# Methodology
+
+This section deals with the development process and how the project
+topic was approached, as well as the decisions made in relation to the
+programming languages and environments.
+
+Furthermore, we deal with the architecture of the application, that is
+divided into stand-alone client area and a server architecture. The data
+exchange and export plays an important role here, which is also
+discussed in this chapter and later in the implementation part of the
+report.
+
+## Project management
+
+Overall, the project took place within five weeks and five participants
+who worked full time on the project. The decisions in the area of
+project management are described in more detail below.
+
```

Comment:
perhaps add here something like that:

Each team member was assigned a role for which he was responsible. The roles were e.g. 'Git responsible', 'Moderator' or 'Scrum master'.